### ISSUE TITLE
Slightly corrupted output when using Bourne shell

### ISSUE THREAD

**Issue by tomfotherby** [Codeception/Codeception#3756]
In my project, I'm switching from Ubuntu running php 5 to Alpine linux running php 7. Alpine uses `sh` (Bourne shell, rather than `bash`) and the codeception output text is a little corrupt:

```bash
$ docker-compose run --rm codecept run tests/unit/models/TagFrequencyMatrixTest

...
 (app\tests\unit\models\TagFrequencyMatrixTest::testGetAdditions) Okitions)... 
 (app\tests\unit\models\TagFrequencyMatrixTest::testGetChanges) Okhanges)... 
```
Previously the output looked better, like this:

```bash
 Test get (app\tests\unit\models\TagFrequencyMatrixTest::testGetAdditions) Ok
 Test get c (app\tests\unit\models\TagFrequencyMatrixTest::testGetChanges) Ok
```

Using `--no-ansi` doesn't help. Using `--no-colors` doesn't help. 

Using `--silent` and `--report` give different, but non-corrupt output:

```bash
app\tests\unit\models\TagFrequencyMatrixTest::testGetAdditions.............Ok
app\tests\unit\models\TagFrequencyMatrixTest::testGetChanges...............Ok
```

### Details

* Codeception version: 2.1.11
* PHP Version: 7.0.13
* Operating System: Alpine Linux 3.4
* Installation type: Composer
* List of installed packages (`composer show`)  too many to show, but `yiisoft/yii2-codeception` is one maybe of interest
* Suite configuration:

```yml
class_name: UnitTester
modules:
    enabled: [Asserts, UnitHelper]

```

**Comment by DavertMik** [comment]
> Codeception version: 2.1.11

Did you try latest version which is 2.2.6? It has more than year of updates and fixes comparing to 2.1.11

**Comment by tomfotherby** [comment]
@DavertMik  -  that's the first thing I should have tried, sorry for wasting your time. Yup it "fixes" it, the output looks fabulous now in bourne shell, colors and everything ;)

```
docker-compose run --rm codecept run tests/unit/models/TagFrequencyMatrixTest
Codeception PHP Testing Framework v2.2.6
Powered by PHPUnit 5.7-g5042a8e by Sebastian Bergmann and contributors.

Unit Tests (2) 
✔ TagFrequencyMatrixTest: Get additions (0.02s)
✔ TagFrequencyMatrixTest: Get changes (0.01s)
```
